ZXing.Net.MAUI/Platforms/Android/CameraManager.android.cs

```diff
@@ -63,6 +65,7 @@ public void Connect()
 				// Frame by frame analyze
 				imageAnalyzer = new ImageAnalysis.Builder()
 					.SetDefaultResolution(new Android.Util.Size(640, 480))
+					.SetOutputImageRotationEnabled(true)
```

Comment:
Shouldn't this be set based on some input?

I don't want it to auto-rotate if I initialise this:
```
          new CameraBarcodeReaderView
          {
            Options = new BarcodeReaderOptions
            {
              AutoRotate = false,
              Multiple = false,
            }
          }
```